### ISSUE TITLE
Unable to load cached images: loading cached images: stat ... kube-scheduler_v1.25.1 ... no such file or directory

### ISSUE THREAD

**Issue by conradwt** [kubernetes/minikube#14960]
### What Happened?

When installing a newer image with

```
minikube start --kubernetes-version=v1.25.1
😄  minikube v1.26.1 on Darwin 12.6 (arm64)
❗  Specified Kubernetes version 1.25.1 is newer than the newest supported version: v1.24.3. Use `minikube config defaults kubernetes-version` for details.
✨  Using the docker driver based on existing profile
👍  Starting control plane node minikube in cluster minikube
🚜  Pulling base image ...
💾  Downloading Kubernetes v1.25.1 preload ...
    > preloaded-images-k8s-v18-v1...:  320.78 MiB / 320.78 MiB  100.00% 39.50 M
🔄  Restarting existing docker container for "minikube" ...
🐳  Preparing Kubernetes v1.25.1 on Docker 20.10.17 ...
❌  Unable to load cached images: loading cached images: stat /Users/conradwt/.minikube/cache/images/arm64/k8s.gcr.io/kube-scheduler_v1.25.1: no such file or directory
🤦  Unable to restart cluster, will reset it: apiserver health: apiserver healthz never reported healthy: cluster wait timed out during healthz check
    ▪ Generating certificates and keys ...
    ▪ Booting up control plane ...
    ▪ Configuring RBAC rules ...
🔎  Verifying Kubernetes components...
    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
🌟  Enabled addons: storage-provisioner, default-storageclass
🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
```

I'm seeing the following issue:

```
Unable to load cached images: loading cached images: stat /Users/conradwt/.minikube/cache/images/arm64/k8s.gcr.io/kube-scheduler_v1.25.1: no such file or directory
```

https://gist.github.com/conradwt/cf89b331a5f7c76688ff097e3ad97968

### Attach the log file

[log.txt](https://github.com/kubernetes/minikube/files/9578739/log.txt)

### Operating System

macOS 12.6

### Driver

Docker Desktop 4.12.0 (85629)

**Comment by afbjorklund** [comment]
Seems like a bug in the cache handling, the image has been moved from k8s.gcr.io to registry.k8s.io

```
Got preloaded images: -- stdout --
registry.k8s.io/kube-apiserver:v1.25.1
registry.k8s.io/kube-proxy:v1.25.1
registry.k8s.io/kube-scheduler:v1.25.1
registry.k8s.io/kube-controller-manager:v1.25.1
k8s.gcr.io/kube-apiserver:v1.24.5
k8s.gcr.io/kube-proxy:v1.24.5
k8s.gcr.io/kube-controller-manager:v1.24.5
k8s.gcr.io/kube-scheduler:v1.24.5
registry.k8s.io/pause:3.8
registry.k8s.io/etcd:3.5.4-0
registry.k8s.io/coredns/coredns:v1.9.3
k8s.gcr.io/etcd:3.5.3-0
k8s.gcr.io/pause:3.7
k8s.gcr.io/coredns/coredns:v1.8.6
k8s.gcr.io/pause:3.6
gcr.io/k8s-minikube/storage-provisioner:v5

-- /stdout --
```

* https://github.com/kubernetes/minikube/issues/14769

**Comment by klaases** [comment]
> Think it was fixed in minikube 1.27.0 ?

Hi @conradwt, can you confirm whether this was fixed as suggested above?

**Comment by kevin-dimichel** [comment]
I just experienced this issue with minikube version v1.27.1

```
$ minikube start
😄  minikube v1.27.1 on Darwin 12.6 (arm64)
✨  Using the docker driver based on existing profile

⛔  Docker Desktop only has 3933MiB available, you may encounter application deployment failures.
💡  Suggestion:

    1. Click on "Docker for Desktop" menu icon
    2. Click "Preferences"
    3. Click "Resources"
    4. Increase "Memory" slider bar to 2.25 GB or higher
    5. Click "Apply & Restart"
📘  Documentation: https://docs.docker.com/docker-for-mac/#resources

👍  Starting control plane node minikube in cluster minikube
🚜  Pulling base image ...
🏃  Updating the running docker "minikube" container ...
🐳  Preparing Kubernetes v1.25.2 on Docker 20.10.18 ...
❌  Unable to load cached images: loading cached images: stat /Users/kevin/.minikube/cache/images/arm64/registry.k8s.io/kube-scheduler_v1.25.2: no such file or directory
    ▪ Generating certificates and keys ...
    ▪ Booting up control plane ...
    ▪ Configuring RBAC rules ...
🔎  Verifying Kubernetes components...
    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
🌟  Enabled addons: default-storageclass, storage-provisioner
🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
```

**Comment by spowelljr** [comment]
The issue @conradwt is experiencing should be fixed with v1.27.0 with https://github.com/kubernetes/minikube/pull/14768.

The issue you're experiencing @kevin-dimichel is something else and I've seen it from time to time. I made an issue to track it https://github.com/kubernetes/minikube/issues/15237 so please follow along there, I'm going to close this issue.
